sdk/communication/communication-network-traversal/src/models.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// delete this pipeline options
+import { PipelineOptions } from "@azure/core-http";
+
+/**
+ * Client options used to configure the CommunicationNetworkTraversal API requests.
+ */
+export interface CommunicationRelayClientOptions extends PipelineOptions {}
+
+/**
+ * A TURN credentials response.
+ */
+export interface CommunicationRelayConfiguration {
```

Comment:
`CommunicationRelayConfiguration` and `CommunicationTurnServer` are already defined in the generated/models.ts, you can just export those.

For renaming `CommunicationTurnCredentialsResponse` to `CommunicationRelayConfiguration` you can add a transform to your swagger .md file. Let me paste the instructions there.

Reply:
I'll leave this for a separate change on a future PR